test/cpp/end2end/grpc_authz_end2end_test.cc

```diff
@@ -798,6 +1120,11 @@ TEST_F(GrpcAuthzEnd2EndTest, FileWatcherRecoversFromFailure) {
   grpc::Status status = SendRpc(channel, &context1, &resp1);
   EXPECT_TRUE(status.ok());
   EXPECT_EQ(resp1.message(), kMessage);
+  ASSERT_EQ(audit_logs_.size(), 1);
```

Comment:
Suggest using `EXPECT_THAT(audit_logs_, ::testing::ElementsAre(...))` here.  That way, if the check fails, we won't bail out early.

Reply:
Done.